renderer/components/upscayl-tab/view/ProgressBar.tsx

```diff
@@ -1,22 +1,26 @@
 import React, { useEffect } from "react";
 import Spinner from "../../icons/Spinner";
 import Logo from "@/components/icons/Logo";
+import NPBC from "@/components/NeonProgressBarComponent"
 
 function ProgressBar({
   progress,
+  progressFloat,
   doubleUpscaylCounter,
   stopHandler,
   batchMode,
 }: {
   progress: string;
+  progressFloat: number;
   doubleUpscaylCounter: number;
   stopHandler: () => void;
   batchMode: boolean;
 }) {
   const [batchProgress, setBatchProgress] = React.useState(0);
-
+  
   useEffect(() => {
     const progressString = progress.trim().replace(/\n/g, "");
+    
     // Remove trailing and leading spaces
```

Comment:
I don't see a point for the use of whitespace here.

Reply:
It makes sense to me. I always do that as well :)

Reply:
well, to me it's just one line so it wouldn't need another linebreak, but yeah the new line is fine. Committing the outdent now